src/NzbDrone.Core/Datastore/Migration/227_update_restrictions_to_release_profiles.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Dapper;
+using FluentMigrator;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(227)]
+    public class update_restrictions_to_release_profiles : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Rename.Table("Restrictions").To("ReleaseProfiles");
+
+            Alter.Table("ReleaseProfiles").AddColumn("Name").AsString().Nullable().WithDefaultValue(null);
+            Alter.Table("ReleaseProfiles").AddColumn("Enabled").AsBoolean().WithDefaultValue(true);
+            Alter.Table("ReleaseProfiles").AddColumn("IndexerId").AsInt32().WithDefaultValue(0);
+            Delete.Column("Preferred").FromTable("ReleaseProfiles");
+
+            Execute.WithConnection(ChangeRequiredIgnoredTypes);
+
+            Execute.Sql("DELETE FROM \"ReleaseProfiles\" WHERE \"Required\" == '[]' AND \"Ignored\" == '[]'");
```

Comment:
`==` should be `=`

Reply:
Hopefully this is the last oversight in the migration process :sweat_smile: